.github/workflows/e2e-tests.yml

```diff
@@ -154,3 +154,72 @@ jobs:
       - name: Install and Run Integration Tests
         run: |
           ./mvnw ${MAVEN_ARGS} -Pitests -pl $IT_MODULE verify
+  openshift-kubernetes-distribution:
+    name: CRC ${{ matrix.crc }} / OKD ${{ matrix.okd }}
+    needs: buildWithoutTests
+    runs-on: ubuntu-latest
+    if: github.repository == 'fabric8io/kubernetes-client'
+    env:
+      SHELL: /bin/bash
+    strategy:
+      fail-fast: false
+      matrix:
+        # OKD version is bound to crc version
+        okd: [v4.14.0]
+        crc: [2.32.0]
```

Comment:
Please add a comment as to why these versions aren't the latest.
Also create an issue for further investigation and future updates and put the link in the comment.

Reply:
I have created this issue https://github.com/fabric8io/kubernetes-client/issues/6415. It's also referenced in TODO comment.